tests/Resources/Page/ODH/AiApps/Rhoam.resource

```diff
@@ -0,0 +1,41 @@
+*** Settings ***
+Documentation       Keyword suites to interact with Anaconda Commercial Edition
```

Comment:
It should mention RHOAM instead of anaconda

Reply:
lol. Changing it